src/instructlab/model/remove.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from pathlib import Path
+import shutil
+import sys
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.configuration import DEFAULTS
+from instructlab.model.backends.backends import is_model_gguf, is_model_safetensors
+
+
+@click.command(name="remove")
+@clickext.display_params
+@click.option(
+    "--model",
+    required=True,
+    help="Specify the model name to remove(check with `ilab model list` first), \n"
+    "e.g. merlinite-7b-lab-Q4_K_M.gguf or instructlab/merlinite-7b-pt.",
+)
+def remove_model(model: str):
+    """Remove model"""
+    models_dir = Path(DEFAULTS.MODELS_DIR)
+    model_path = models_dir / model
+    # Check if model path exists
+    try:
+        if model_path.exists():
+            if model_path.is_file() and is_model_gguf(model_path):
+                print(f"==> Removing model file: {model}")
+                model_path.unlink()
+                print(f"==> Model file {model} has been removed.")
+                sys.exit(0)
+            elif model_path.is_dir() and is_model_safetensors(model_path):
+                print(f"==> Removing model directory: {model}")
+                shutil.rmtree(model_path)
+                print(f"==> Model directory {model} has been removed.")
+                sys.exit(0)
+            else:
+                print(f"Error: {model} is not a valid .gguf file or model directory.")
```

Comment:
... safetensors model directory

Reply:
Should this reference the full path as well.

Model found at {model_path} is not ...